### ISSUE TITLE
Main scene not shown

### ISSUE THREAD

**Issue by optozorax** [t-veor/hypervis#2]
On Windows: 

![image](https://user-images.githubusercontent.com/18114381/85919578-96090480-b896-11ea-8ce4-9b645a036674.png)

On linux:

![image](https://user-images.githubusercontent.com/18114381/85919573-87225200-b896-11ea-8968-5d529cd4217d.png)

Trying to `cargo update` and change `Cargo.toml` like this:

```diff
diff --git a/Cargo.toml b/Cargo.toml
index 7b9c1dd..782f48d 100644
--- a/Cargo.toml
+++ b/Cargo.toml
@@ -5,7 +5,7 @@ authors = ["Tyler Zhang <tianlinzhang@gmail.com>"]
 edition = "2018"
 
 [dependencies]
-winit = "0.20"
+winit = "0.21"
 wgpu = "0.4"
 anyhow = "1.0"
 glsl-to-spirv = "0.1"
```

It compiles, but work the same.

Video card: nvidia gtx 650 ti.

**Comment by t-veor** [comment]
Hey, sorry for taking a while to get around to this. It looks like there was an issue with the previous version of wgpu-rs and a new nVidia graphics driver that caused the imgui overlay to clear the screen. I'm not sure what the cause is, but I've upgraded wgpu to 0.5 and this seems to have fixed the issue. Let me know if the latest master branch works for you.

(The issue was weird - I captured a trace with renderdoc that showed that the imgui render pass was being initialised with vkCmdBeginRenderPass(Load), but it was clearing the screen somehow. I've no idea what was causing this.)
